_dashboards/get-started/quickstart-dashboards.md

```diff
@@ -0,0 +1,117 @@
+
+# Exploring and inspecting data
+
+In [**Discover**](discover/index.md), you can: 
+
+- Choose data to explore, set a time range for that data, search it using [Dashboards Query Language (DQL)]({{site.url}}{{site.baseurl}}/dashboards/dql/), and filter the results.
+- Explore the data, view individual documents, and create tables summarizing the data's contents.
+- Visualize your findings.
+
+## Try it: Getting familiar with Discover
+
+1. On the OpenSearch Dashboards **Home** page, choose **Discover.**
+1. Change the [time filter](time-filter.md) to **Last 7 days**, as shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/last-7--days.png" alt="Time filter interface" width="350"/>
+
+2. Search using the DQL query `FlightDelay:true AND DestCountry: US AND FlightDelayMin >= 60` and then choose **Update**. You should see results for US-bound flights delayed by 60 minutes or more, as shown in the following image.
+   
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/dql-search-field.png" alt="DQL search field example" width="1000"/>
+
+3. To filter data, choose **Add filter** and then select an **Available field**. For example, select `FlightDelayType`, **is**, and **Weather delay** from the **Field, Operator, and Value** dropdown lists, as shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/filter-data-discover.png" alt="Filter data by FlightDelayType field" width="350"/>
+
+# Visualizing data
+
+Raw data can be difficult to comprehend and use. Data visualizations help you prepare and present data in a visual form. In **Dashboard** you can:
+
+- Display data in a single view.
+- Build dynamic dashboards.
+- Create and share reports.
+- Embed analytics to differentiate your applications.
+
+## Try it: Getting familiar with Dashboard
+
+1. On the OpenSearch Dashboards **Home** page, choose **Dashboard**.
+1. Choose **[Flights] Global Flight Data** in the **Dashboards** window, as shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboard-flight-quickstart.png" alt="Data visualization dashboard" width="800"/>
+
+1. To add panels to the dashboard, choose **Edit** and then **Add** from the toolbar.
+1. In the **Add panels** window, choose the existing panel **[Flights] Delay Buckets**. You'll see a pop-up window on the lower right confirming that you've added the panel.
+1. Select `x` to close the Add panels window.
+1. View the added panel **[Flights] Delay Buckets**, which is added as the last panel on the dashboard, as shown in the following image.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/add-panel.png" alt="Add panel to dashboard" width="600"/>
+
+## Try it: Creating a visualization panel
+
+Continuing with the preceding dashboard, you'll create a bar chart comparing the number of canceled flights and delayed flights to delay type and then add the panel to the dashboard.
```

Comment:
Replace the terminating period with a colon.

Reply:
Change made